examples/flutter/example/Earthfile

```diff
@@ -0,0 +1,11 @@
+VERSION 0.8
+
+IMPORT github.com/input-output-hk/catalyst-ci/earthly/flutter:feat-add-guide-for-flutter-dart AS flutter-ci
+
+# Set up the CI environment for Flutter app.
+builder:
+    DO flutter-ci+SETUP
+
+ # Bootstrap the Flutter app if mroject uses melos.
+bootstrap:
+    DO flutter-ci+BOOTSTRAP
```

Comment:
```suggestion
    FROM +builder
    DO flutter-ci+BOOTSTRAP
```

Reply:
Use the builder that was setup,  otherwise you have no way to get files like melos.yml into the image.

This could be simplified all inside the `builder:` target as.

```
builder:
    DO flutter-ci+SETUP

    # Copy stuff here, and customize as necessary.

    DO flutter-ci+BOOTSTRAP
```